Mindbox.Data.Linq/DataContext.cs

```diff
@@ -937,6 +937,18 @@ private void Init(object connection, MappingSource mapping)
             InitTables(this);
         }
 
+        public object CloneWithSameConnection()
+        {
+	        var clonedDataContext = (DataContext) Activator.CreateInstance(GetType());
```

Comment:
```suggestion
        public DataContext CloneWithSameConnection()
        {
	        var clonedDataContext = Create();
```
Create - виртуальный мутод, который перегружается в MindboxDataContext